api/core/model_runtime/model_providers/openai_api_compatible/llm/llm.py

```diff
@@ -120,6 +120,8 @@ def validate_credentials(self, model: str, credentials: dict) -> None:
             endpoint_url = credentials["endpoint_url"]
             if not endpoint_url.endswith("/"):
                 endpoint_url += "/"
+            if not endpoint_url.endswith("v1/"):
+                endpoint_url += "v1/"
```

Comment:
Hi, It's not recommended to hard code path "v1" into `openai_api_compatible`, as it's not backwards compatible, all credentials already saved before will be appended with "v1", once it is originally it does not need "v1" as a prefix, their providers may not work as before.

Reply:
@Yeuoly
thank you for advice.
I reverted that part of code.